x-pack/plugins/fleet/public/components/custom_assets_accordion.tsx

```diff
@@ -31,11 +31,13 @@ export interface CustomAssetsAccordionProps {
     description: string;
   }>;
   initialIsOpen?: boolean;
+  title?: string;
 }
 
 export const CustomAssetsAccordion: FunctionComponent<CustomAssetsAccordionProps> = ({
   views,
   initialIsOpen = false,
+  title = undefined,
```

Comment:
Don't need to set an optional prop with the default as `undefined`, it can be just `title`